src/main/java/com/datatorrent/netlet/util/VarInt.java

```diff
@@ -59,6 +61,18 @@ public static int write(int value, byte[] buffer, int offset, int size)
     return i;
   }
 
+  public static int write(int value, ByteBuffer buffer)
```

Comment:
It is used to write length to the direct buffer in the WriteOnlyLengthPrependerClient